docs/developer_guide.md

```diff
@@ -96,14 +96,11 @@ Run the following code to start mixer gRPC server
 # cd into repo root directory
 
 go run cmd/main.go \
+    --mixer_project=datcom-mixer-staging \
+    --store_project=google.com:datcom-store-dev \
```

Comment:
nit: TODO: to switch this to datcom-store ?

Reply:
Done